PomBndRepo: Exception

When doing right-click on a RepositoryBundle entry in Repositories view:

```
java.lang.RuntimeException: java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at bndtools.views.repository.RepositoriesView.lambda$5(RepositoriesView.java:1005)
	at org.eclipse.jface.action.MenuManager.fireAboutToShow(MenuManager.java:338)
	at org.eclipse.jface.action.MenuManager.handleAboutToShow(MenuManager.java:468)
	at org.eclipse.jface.action.MenuManager$2.menuShown(MenuManager.java:495)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:272)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:91)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4660)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1622)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1645)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1626)
	at org.eclipse.swt.widgets.Menu.menuWillOpen(Menu.java:803)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:6381)
	at org.eclipse.swt.internal.cocoa.OS.objc_msgSend(Native Method)
	at org.eclipse.swt.internal.cocoa.NSMenu.popUpContextMenu(NSMenu.java:80)
	at org.eclipse.swt.widgets.Menu._setVisible(Menu.java:273)
	at org.eclipse.swt.widgets.Display.runPopups(Display.java:4504)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3996)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:648)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:555)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:173)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:208)
	at org.bndtools.applaunch.BndApplicationLauncher.lambda$0(BndApplicationLauncher.java:34)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at aQute.launcher.Launcher.handleMainCallable(Launcher.java:464)
	at aQute.launcher.Launcher.launch(Launcher.java:437)
	at aQute.launcher.Launcher.run(Launcher.java:185)
	at aQute.launcher.Launcher.main(Launcher.java:161)
	at aQute.launcher.pre.EmbeddedLauncher.executeWithRunPath(EmbeddedLauncher.java:169)
	at aQute.launcher.pre.EmbeddedLauncher.findAndExecute(EmbeddedLauncher.java:134)
	at aQute.launcher.pre.EmbeddedLauncher.main(EmbeddedLauncher.java:51)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at aQute.bnd.repository.maven.pom.provider.BndPomRepository.actions(BndPomRepository.java:493)
	at bndtools.views.repository.RepositoriesView.lambda$5(RepositoriesView.java:981)
	... 35 more
```

fix coming.

--- a/biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/BndPomRepository.java
+++ b/biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/BndPomRepository.java
@@ -482,19 +482,23 @@ public class BndPomRepository extends BaseRepository
 			cb.copy(gavs);
 		});
 
-		String bsn = (String) target[0];
-		Version version = (Version) target[1];
-		Archive archive = getArchiveByBsnVersion(bsn, version);
 
-		RepoActionsUtil.addSources(archive, (a) -> {
+		if (target.length == 2) {
+			// Revision Action: 0 = bns, 1 = version
+			String bsn = (String) target[0];
+			Version version = (Version) target[1];
+			Archive archive = getArchiveByBsnVersion(bsn, version);
 
-			try {
-				return repoImpl.getMavenRepository()
-					.get(a);
-			} catch (Exception e) {
-				throw Exceptions.duck(e);
-			}
-		}, menu);
+			RepoActionsUtil.addSources(archive, (a) -> {
+
+				try {
+					return repoImpl.getMavenRepository()
+						.get(a);
+				} catch (Exception e) {
+					throw Exceptions.duck(e);
+				}
+			}, menu);
+		}
 
 		return menu;
 	}

